dashboard/app/models/pd/application/facilitator1920_application.rb

```diff
@@ -98,10 +98,10 @@ def on_successful_create
     def self.fit_cohort(applications_query = all)
       applications_query.
```

Comment:
Could a comment be helpful here to explain what this query is looking for?

Reply:
updated the comment!